crates/server/src/systems.rs

```diff
@@ -78,5 +81,13 @@ fn tick_clients(_game: &mut Game, server: &mut Server) -> SysResult {
     for client in server.clients.iter() {
         client.tick();
     }
+
+    static COUNTER: std::sync::atomic::AtomicUsize = AtomicUsize::new(0);
+    if COUNTER.fetch_add(1, std::sync::atomic::Ordering::SeqCst) >= 200 {
+        for client in server.clients.iter() {
```

Comment:
Looks like this commit got in here again :)

Reply:
I think its a compelling feature!